src/main.cpp

```diff
@@ -23,6 +24,19 @@ SolarGators::DataModules::OrionBMSRx3 OrionBMSRx3(SolarGators::DataModuleInfo::B
 SolarGators::DataModules::OrionBMSRx4 OrionBMSRx4(SolarGators::DataModuleInfo::BMS_RX4_MSG_ID, 0);
 SolarGators::DataModules::OrionBMSRx5 OrionBMSRx5(SolarGators::DataModuleInfo::BMS_RX5_MSG_ID, 0);
 
+//registering mppt data modules
+//there are several more modules than this, but i only want to register essential info from each
+SolarGators::DataModules::Mpptx0 MPPT0_Rx_0(SolarGators::DataModuleInfo::MPPT0_MSG_ID);
+SolarGators::DataModules::Mpptx0 MPPT1_Rx_0(SolarGators::DataModuleInfo::MPPT1_MSG_ID);
+SolarGators::DataModules::Mpptx0 MPPT2_Rx_0(SolarGators::DataModuleInfo::MPPT2_MSG_ID);
+
+SolarGators::DataModules::Mpptx1 MPPT0_Rx_1(SolarGators::DataModuleInfo::MPPT0_MSG_ID + 1);
+SolarGators::DataModules::Mpptx1 MPPT1_Rx_1(SolarGators::DataModuleInfo::MPPT1_MSG_ID + 1);
+SolarGators::DataModules::Mpptx1 MPPT2_Rx_1(SolarGators::DataModuleInfo::MPPT2_MSG_ID + 1);
+
+SolarGators::DataModules::Mpptx5 MPPT0_Rx_5(SolarGators::DataModuleInfo::MPPT0_MSG_ID + 5);
+SolarGators::DataModules::Mpptx5 MPPT1_Rx_5(SolarGators::DataModuleInfo::MPPT1_MSG_ID + 5);
+SolarGators::DataModules::Mpptx5 MPPT2_Rx_5(SolarGators::DataModuleInfo::MPPT2_MSG_ID + 5);
```

Comment:
It might be easier to read if seperate enums are used for each message + mppt number combination. My brain is not happy when translating them between here and in the data modules

https://github.com/Solar-Gators/CAN-Decoder/blob/b0a402a6029ca2c525ea617aef4f8772d8061623/src/DataModules/src/Mppt.cpp#L61-L69

Reply:
ie.

MPPT0_RX0_MSG_ID = MPPT0_BASE_MSG_ID
MPPT0_RX1_MSG_ID = MPPT0_BASE_MSG_ID + 1

Reply:
That makes sense, will do! Thanks!

Reply:
How's everything looking now?